resources/build/zip.inc.sh

```diff
@@ -23,13 +23,32 @@ function zip_files() {
   shift
 
   # Parse rest of parameters
-  FLAGS=()
-  INCLUDE=()
+  local ZIP_FLAGS=()
+  local SEVENZ_FLAGS=('a') # 7z requires a command
+  local INCLUDE=()
   while [[ $# -gt 0 ]] ; do
     case "$1" in
+      -r)
+        # Common flags to zip and 7z
+        ZIP_FLAGS+=($1)
+        SEVENZ_FLAGS+=($1)
+        shift
+        ;;
+      -1)
+        # Compression level where -1 indicates fastest compression speed
+        ZIP_FLAGS+=($1)
+        SEVENZ_FLAGS+=("-mx1")
+        shift
+        ;;
+      -9)
```

Comment:
zip compression level can go from `-1` (fastest) to `-9` (slowest)
7z compression level can go from `-mx1` (fastest) to `-mx9` (slowest)

Is there some Bash-fu to regex these range of digits?

Reply:
How about:

```sh
   -[0123456789])
```